app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -77,11 +72,11 @@ fun WiFiIconText() {
         ),
             contentDescription = "WiFi Icon",
             modifier = Modifier
-                .size(30.dp)
+                .size(30.dp),
         )
         Text(
             text = "Wi-Fi",
-            fontSize = (26.sp)
+            fontSize = (26.sp),
```

Comment:
Added comma as per the kotlin best practices
https://kotlinlang.org/docs/coding-conventions.html